assets/js/handlebars/templates/versions-dropdown.handlebars

```diff
@@ -6,4 +6,7 @@
       </option>
     {{/each}}
   </select>
+  {{#unless nodes.0.isCurrentVersion}}
+    <button type="button" class="sidebar-projectLatestVersionButton" value="{{nodes.0.version}}">Jump to Latest</button>
+  {{/unless}}
```

Comment:
I believe there is an edge case that would work unexpectedly. Say we used this to render Elixir 1.14.4 docs. When there will be 1.15.0-rc.0 it would be at the top of the list, `nodes.0`, but it should _not_ be considered _latest_.

This would definitely be an issue for Postgrex:

<img width="726" alt="image" src="https://github.com/elixir-lang/ex_doc/assets/76071/4edaded3-c506-4b35-8044-fd1321d5a873">

There's an old issue for hex docs around this https://github.com/hexpm/hexdocs/issues/10, btw.

Reply:
Oof, Did not realize that.

Would it be alright to add a `latestVersionNode` or `latestVersionUrl` variable to `docs_config.js` to provide the relevant information?
Would need to update hexdocs to add that variable as well, but thats easy enough.

That would allow explicit selection of which docs are latest, while still allowing pre-releases in the list.

Reply:
Variable sounds good to me. Alternatively we could add `"latest":true` pair like this; before:

```js
[
  {"url":"https://hexdocs.pm/foo/1.0.0","version":"v1.0.0"},
  {"url":"https://hexdocs.pm/foo/0.1.0","version":"v0.1.0"}
]
```

after:

```js
[
  {"url":"https://hexdocs.pm/foo/1.0.0","version":"v1.0.0","latest":true},
  {"url":"https://hexdocs.pm/foo/0.1.0","version":"v0.1.0"}
]
```

The advantage is if this key is not defined, for packages which haven't been published recently, we can gracefully handle that as opposed to getting undefined variable error in JS. But maybe it's fine like you proposed. cc @josevalim @ericmj